Resources/public/js/views/ez-dashboardblockbasedview.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockbasedview', function (Y) {
+    'use strict';
+    /**
+     * Provides the Dashboard Block Based View class
+     *
+     * @module ez-dashboardblockbasedview
+     */
+    Y.namespace('eZ');
+
+    var SELECTOR_CONTENT = '.dashboard-content';
+
+    /**
+     * The dashboard view
+     *
+     * @namespace eZ
+     * @class DashboardBlockBasedView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.DashboardBlockBasedView = Y.Base.create('dashboardBlockBasedView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            this.onceAfter('activeChange', function (event) {
```

Comment:
yes I just realized, I misread the following line :)
But I think we should not try to do that this way. We should only cover the case where the block are added before the rendering of the dashboard (ie what happens when you add the block in the initializer of a plugin). I don't see any usecase where you would need to add a block after that point.